lib/errors/generic_server_exception.dart

```diff
@@ -0,0 +1,8 @@
+class GenericServerException implements Exception {
+  @override
+  String toString() {
+    return message;
+  }
+
+  String get message => 'Something went wrong';
```

Comment:
You can return this string directly into `toString`, there's no need to define a getter. I'm sorry if I did that before, it's probably technical debt.